### ISSUE TITLE
Auth Emulator UI crashes when there is no Provider Info

### ISSUE THREAD

**Issue by samtstern** [firebase/firebase-tools-ui#381]
I created a user like this:
```js
  const user = await admin.auth().createUser({
    uid: "uid" + new Date().getTime(),
    displayName: "Sam User"
  });
```

And then when I load the emulator UI it complains about `map of undefined`

I think it's this bit:
```jsx
      <div className={styles.iconWrapper}>
        {user.providerUserInfo.map(
          providerInfo =>
            providerToIconMap[providerInfo.providerId] && (
              <Icon
                icon={providerToIconMap[providerInfo.providerId]}
                key={providerInfo.providerId}
                aria-label={providerInfo.providerId}
              />
            )
        )}
```

**Comment by samtstern** [comment]
@yuchenshi not clear if this is a UI bug or if the emulator is not handling `createUser` properly.  I tried adding `email` to the create request but it didn't change anything.

**Comment by yuchenshi** [comment]
Fixed by #382. Release coming soon.
